v2.0/inverted-indexes.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Inverted Indexes
+summary: Inverted indexes improve your database's performance and usefulness by helping SQL locate schemaless data.
+toc: false
+toc_not_nested: true
```

Comment:
Let's remove this so the h3 headers show up.

Reply:
done